examples/units/bar_unit_demo.py

```diff
@@ -1,3 +1,13 @@
+"""
+=========================
+Group barchart with units
+=========================
+
+This is the same example as
+<a href='http://matplotlib.org/examples/api/barchart_demo.html'>
```

Comment:
I don't think this works. Once we'll finish the migration to sphinx gallery, rst markup will work.